opencsp/app/sofast/test/test_SystemSofastFringe.py

```diff
@@ -1,20 +1,49 @@
 """Unit test suite to test the System class
 """
 
+import numpy as np
 import os
 import unittest
 
 import pytest
 
+import opencsp.app.sofast.lib.ImageCalibrationGlobal as icg
+import opencsp.app.sofast.lib.ImageCalibrationScaling as ics
 from opencsp.app.sofast.lib.Fringes import Fringes
 from opencsp.app.sofast.lib.SystemSofastFringe import SystemSofastFringe
 from opencsp.app.sofast.test.ImageAcquisition_no_camera import ImageAcquisition
 from opencsp.common.lib.deflectometry.ImageProjection import ImageProjection
+import opencsp.common.lib.deflectometry.test.test_ImageProjection as test_ip
 from opencsp.common.lib.opencsp_path.opencsp_root_path import opencsp_code_dir
+import opencsp.common.lib.tool.exception_tools as et
+import opencsp.common.lib.tool.file_tools as ft
 
 
+@pytest.mark.no_xvfb
 class TestSystemSofastFringe(unittest.TestCase):
-    @pytest.mark.no_xvfb

```

Comment:
Does this really not need to be marked no_xvfb?

Reply:
I moved this decorator up to apply to whole class. Is that not correct?